tests/acceptance/features/apiSpaces/favorite.feature

```diff
@@ -15,7 +15,7 @@ Feature: favorite
     And user "Brian" has accepted share "/PARENT" offered by user "Alice"
     When user "Brian" favorites element "/PARENT" in space "Shares Jail" using the WebDAV API
     Then the HTTP status code should be "207"
-    And as user "Brian" folder "/PARENT" inside space "Shares Jail" should contain a property "oc:favorite" with value "1"
+    And as user "Brian" folder "/PARENT" inside space "Shares Jail" should be favorited
```

Comment:
if we already have a step that satisfies the scenario condition why add a new step?

Reply:
It does the same and the step also makes more sense that the resource has been favorited.

Reply:
if we go for grammar then this is nice. But if we want to reduce the line of code in ocis, it impacts that. The point to note is whether should we have to choose grammar or reduce lines of code.

Reply:
@amrita-shrestha this step is actually reusing the code which is already in core.

Reply:
IMO `should be favorited` is better than `should contain a property "oc:favorite" with value "1"` because the former is more readable and understandable even from a normal person's point of view.